README.md

```diff
@@ -170,6 +171,24 @@ This optional generator, not included with the default generator, creates the ap
 
 The last group of generators are for your convenience if you want to start overriding code included as part of the Rails engine. For example by default the engine provides a simple SessionController, if you run the `rails generate shopify_app:controllers` generator then this code gets copied out into your app so you can start adding to it. Routes and views follow the exact same pattern.
 
+Deploying to a Subdirectory
+---------------------------
```

Comment:
I prefer just `Mounting the Engine`. Its descriptive enough that users won't read the section unless they are trying to mount it in a different way and I think the title is descriptive enough to help users who are looking for sub directory mounting to find what they need